bin/Phantom/cards/examples/phantomProduction_example.cfg

```diff
@@ -0,0 +1,177 @@
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+#                                                                                        #
+#           template cfg file for the usage of the phantom generator in CMS              #
+#                                                                                        #
+# see https://twiki.cern.ch/twiki/bin/view/CMS/PhantomGeneratorCMS for more details      #
+#                                                                                        #
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+
+
+[general]
+# where the zipped version of the compiled phantom code can be found
+# and where the gridpacks will be created
+
+# CMSSW environment for the gridpack creation
+CMSSW = CMSSW_9_2_6_patch1
+
+# phantom version to be used
+package = https://cms-project-generators.web.cern.ch/cms-project-generators/phantom/phantom_1_3_slc6_amd64_gcc530.tar.gz
+# this page should give a reasonable association between architecture and CMSSW release:
+# https://cmssdt.cern.ch/SDT/cgi-bin/ReleasesXML?anytype=1
+ARCH = slc6_amd64_gcc530
+
+# phantom r.in file to be used (provided in the genproduction github)
+# this is expected to be found in the same folder from where the submission script
+# is launched.
+# If not found, the one in the phantom release is used
+# template = template_phantom_1_2_8_nc1_lowopt_r.in
+
+# folder name containing the gridpack, the gridpack name will be
+# foldername.tgz
+foldername = gridpackTesting_SS
+
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+[submission]
+# parameters for the job submission 
+# (for the time being with LSF oron LXPLUS, CONDOR being tested)
+scheduler = LSF
+queue     = 1nw
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+[generation]
+# what generation has to be done, with which configuration parameters
+# production channel, the list of leptons is expected
+channel            = mu e vm_ ve_
+# channel            = mu e_ vm_ ve
+
+# number of top/antitop quarks required in the
+# final state. If the option is not set any number of
+# tops is accepted. If the option is of the form
+# n+ with n an integer any reaction with at least
+# n tops are accepted.
+# -1 means no restrictions
+topnumber          = -1
+
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+
+[parameters]
+# parameters for the phantom config file
+# parameters in this cfg file will be replaced in the phantom cfg template.
+
+# NECESSARY PARAMETER
+# the full list of available pdf is found in the folder stored 
+# in the $LHAPDF_DATA_PATH environment variable of the CMSSW release setup
+# working with LHAPDF6 only
+PDFname        = NNPDF30_nnlo_as_0118
```

Comment:
this should be NNPDF31_nnlo_hessian_pdfas (306000)

Reply:
ok

Reply:
(how do I change something before the merge, in this PR? do I have to delete the PR and make a new one, or what???)

Reply:
cool, I managed to ;) the pdfs and cmssw release are in the cfg file

Reply:
+1